### PR TITLE
[FIX] base: don't unlink base.open_menu but reset it

### PR DESCRIPTION
**Pull request by nle-odoo**
The ir.actions.todo record base.open_menu is used to redirect the user
to a menu after installing a module.

To do so, a new module override the record changing its action (toward a
menu) and setting its state to "open".

But with f7fe753 when one such changed action is removed after
uninstalling a module (or removing the action by any other way) the
base.open_menu would also be removed. Thus thereafter the open_menu will
not work anymore (since type "automatic" is lost).

With this commit, when whenever base.open_menu is removed, instead its
action will be replaced by base.action_client_base_menu which is the
original action.

related to: #11216, opw-660269

**pr note:**

tl:dr when uninstalling a module, removing an action set on `base.open_menu` todo would remove the todo itself, but it is not wanted.

ugly but the alternative I had was: https://github.com/odoo-dev/odoo/commit/0872203

rco also suggested to reset the `open_menu` before module installation but it is a same (or higher) level of hackiness

### CODE REVIEW

**Comment by nle-odoo** [comment]
merged in https://github.com/odoo/odoo/commit/4309bd3
